doc/source/graph/protocols.md

```diff
@@ -83,6 +83,6 @@ In particular,
 | -- | -- | -- |
 | [TRITON_SERVER](../servers/triton.md) | ✅ | [NVIDIA Triton](https://github.com/triton-inference-server/server) |
 | [SKLEARN_SERVER](../servers/sklearn.md) | ✅  | [Seldon MLServer](https://github.com/seldonio/mlserver) |
-| [XGBOOST_SERVER](../servers/xgboost.md) | ✅  | [Seldon MLServer](https://github.com/seldonio/mlserver) |
+| [XGBOOST_SERVER](../servers/xgboost.md) | ✅  | [Seldon MLFlow](../servers/mlflow.md) |
```

Comment:
Shouldn't we keep the `XGBOOST_SERVER` entry on the table as well?

Reply:
good spot I messed up the docs. done now